bin/git-coauthor

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
```

Comment:
My bad. The `-u` option raises out`$1: unbound variable` error when `coauthor` argument is missing. We have already checked the argument explicitly and print appropriate error message, so the error is not useful.
Could you remove the `-u` option? Thanks!